<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ideal Spectra-Generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Number of Components:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LNumberOfComponents"/>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Number of Counts:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LNumberOfCounts"/>
          </item>
          <item>
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>Spectrum Range (ns)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LSpectrumRange"/>
          </item>
          <item>
           <widget class="QLabel" name="label_28">
            <property name="text">
             <string>Binning (ps)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LBinning"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>16</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Spektrum</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="ContiniousLifetime">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Continious Lifetime</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Lifetime 1 (ns):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LLifetime1"/>
            </item>
            <item>
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>Sigma 1:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LSigma1"/>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Intensity 1:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LIntensity1"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Lifetime 2 (ns):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LLifetime2"/>
            </item>
            <item>
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>Sigma 2:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LSigma2"/>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Intensity 2:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LIntensity2"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Lifetime 3 (ns):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LLifetime3"/>
            </item>
            <item>
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>Sigma 3:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LSigma3"/>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Intensity 3:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LIntensity3"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Lifetime 4 (ns):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LLifetime4"/>
            </item>
            <item>
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string>Sigma 4:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LSigma4"/>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Intensity 4:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LIntensity4"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Lifetime 5 (ns):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LLifetime5"/>
            </item>
            <item>
             <widget class="QLabel" name="label_26">
              <property name="text">
               <string>Sigma 5:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LSigma5"/>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Intensity 5:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LIntensity5"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="font">
             <font>
              <family>Arial Black</family>
              <pointsize>16</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>IRF</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>FWHM 1 (ns):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LFWHM1"/>
            </item>
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Intensity 1:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LFWHM_Intensity1"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>FWHM 2 (ns):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LFWHM2"/>
            </item>
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Intensity 2:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LFWHM_Intensity2"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>t0 shift of FWHM 2:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Lt0_shift"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_20">
            <property name="font">
             <font>
              <family>Arial Black</family>
              <pointsize>16</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Background</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>BG:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LBackground"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>948</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBack">
    <property name="title">
     <string>Optionen</string>
    </property>
    <addaction name="actionStart_Window"/>
   </widget>
   <addaction name="menuBack"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStart_Window">
   <property name="text">
    <string>Back</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
